docs/r2dbc.md

```diff
@@ -157,20 +157,23 @@ username.
 * Postgres: If the IAM email ends with `.gserviceaccount.com`, remove
   the `.gserviceaccount.com` suffix from the email.
 
-For example, if the full IAM user account
-is `my-sa@my-project.iam.gserviceaccount.com`,
-You would update the code, replacing these examples with the appropriate values
-for your database engine:
+For example, if the full IAM user account is 
+`my-sa@my-project.iam.gserviceaccount.com`, then the shortened database username
+would be `my-sa` for Mysql, and `my-sa@my-project.iam` for Postgres. 
+
+**Note:** a non-empty string value for the `password` property must be set.
+While this property will be ignored when connecting with the Cloud SQL Connector
+using IAM auth, leaving it empty will cause driver-level validations to fail.
+
+#### Example
+
+Replace these parameters in the example based on your database type:
 
 |               | Mysql | Postgres             |
 |---------------|-------|----------------------|
 | <PROTOCOL>    | mysql | postgresql           |  
```

Comment:
FYI this table doesn't render correctly (same for the R2DBC docs).

Reply:
![image](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/assets/1175430/1ae254bc-aa3c-479e-9f39-e713fda99f1a)

Reply:
Fixed.